controllers/squad/finder.js

```diff
@@ -0,0 +1,41 @@
+const { Op } = require('sequelize');
+const { Warehouse, Sticker, team } = require('../../databases/db');
```

Comment:
```suggestion
const { Warehouse, Sticker, Team } = require('../../databases/db');
```
Modelos en mayuscula pls

Reply:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAH